### PR TITLE
Updates to flatpak package

### PR DESCRIPTION
**Pull request by perezale**
Hi team!

Let me thank you for all your help this past week. We've recently gone through the verification process and have been verified too.

Now, we are interested in making the following updates:

- Following tests conducted by our development team, I've been tasked with elevating permissions and granting write access across the user's home folder. This request stems from the need to support a feature called "Switch Workspace," allowing users to change the working directory location. We believe that setting host:create would be the most suitable solution for our use case. Could you please provide further insight on this matter? Our initial encounter with this issue was when it was identified as a bug in openSUSE, preventing the creation of the workspace directory, regardless of our previous setting `--persist`. 

As usual, many thanks for your help!

EDIT: We'll move this to another PR
- We would like to improve the meta file linking, we have now pushed all flatpak assets to the repository instead of our public file server.

### CODE REVIEW

File: com.scanoss.sbom-workbench.yml [hunk 1]
```diff
@@ -36,10 +34,10 @@ modules:
         url: https://github.com/scanoss/sbom-workbench/releases/download/v1.12.4/sbom-workbench-1.12.4-linux-x86_64.AppImage
         sha256: 53f9dcb75e837d76199e8d8f68ae2e262a61ab36a61a9aa7d8bd7899f77dc751
       - type: file
-        url: https://download.scanoss.com/sbom-workbench/flathub/com.scanoss.sbom-workbench.metainfo.xml
-        sha256: dbc3b0914f7587f3432d9b0dac2201e5a0defc3a18496ec87086d596f5035086
+        url: https://raw.githubusercontent.com/scanoss/sbom-workbench/main/.erb/flatpak/com.scanoss.sbom-workbench.metainfo.xml
```
Comment (hfiguiere) [inline com.scanoss.sbom-workbench.yml]: by linking to a branch you just ensure that the build WILL break next time there is a change.

File: com.scanoss.sbom-workbench.yml [hunk 2]
```diff
@@ -13,9 +13,7 @@ finish-args:
   - "--socket=pulseaudio"
   - "--share=network"  
   - "--device=dri"    
-  - "--persist=.scanoss"
-  - "--persist=scanoss-workspace"
-  - "--persist=.pki"
+  - "--filesystem=host:create"
```
Comment (hfiguiere) [inline com.scanoss.sbom-workbench.yml]: what does `host:create` mean? Also you mention `home` but put `host`.

File: com.scanoss.sbom-workbench.yml (hunk 1 above)
Comment (perezale) [reply com.scanoss.sbom-workbench.yml]: You're right, we will improve this. We will wait until our next release to merge the asset relocation.

File: com.scanoss.sbom-workbench.yml (hunk 2 above)
Comment (perezale) [reply com.scanoss.sbom-workbench.yml]: thanks for catching it

**Comment by perezale** [comment]
Hello! Thank you for sharing, @hfiguiere . We've reviewed your comments and will wait until our next release to merge the asset relocation. 

However, we still plan to merge the filesystem fix for tomorrow's event.

We have changed the permission to `home:rw`.  Let us know your thoughts.

**Pull request merged**